easybuild/easyconfigs/x/X11/X11-20190311-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,212 @@
+easyblock = 'Bundle'
+
+name = 'X11'
+version = '20190311'
+
+homepage = 'https://www.x.org'
+description = "The X Window System (X11) is a windowing system for bitmap displays"
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = [
+    XORG_LIB_SOURCE,
+    XORG_PROTO_SOURCE,
+    'http://xcb.freedesktop.org/dist/',
+    'http://xkbcommon.org/download/',
+    'http://cgit.freedesktop.org/xorg/util/macros/snapshot',
+    XORG_DATA_SOURCE + '/xkeyboard-config',
+    XORG_DATA_SOURCE,
+]
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('fontconfig', '2.13.1'),
+    ('freetype', '2.9.1'),
+    ('zlib', '1.2.11'),
+]
+builddependencies = [
+    ('binutils', '2.31.1'),
+    ('Autotools', '20180311'),
+    ('Bison', '3.0.5'),
```

Comment:
Why is this old version of Bison required?

Reply:
Sorry, my bad, didn't notice the toolchain

Reply:
It's not required, it's good enough, and we don't have easyconfigs for newer `Bison` versions yet that use `GCCcore/8.2.0`